### ISSUE TITLE
Request Denied

### ISSUE THREAD

**Issue by brbeaird** [pfeffed/liftmaster_myq#6]
Seems like Chamberlain may have totally shut down the myqexternal.myqdevice.com endpoint. I get Request Denied for a simple auth call.

**Comment by brbeaird** [comment]
MyQ made significant changes to the auth call. It now uses POST and requires additional headers. Details can be found here: http://forum.universal-devices.com/topic/17486-python-control-of-myq-garage-doors/page-4#entry205234

**Comment by EnGlamdring** [comment]
@brbeaird so I'm assuming you've got it working?

**Comment by brbeaird** [comment]
Yes, it's working great now. This is what I did on my own integration: https://github.com/brbeaird/SmartThings_MyQ/commit/3790c9daf6a8769d16935a4b6f911e923e3bf51c

**Comment by fischgeek** [comment]
Can anyone assist me with tracking down the MyQApplicationId value? I can't seem to find it and I get the following error when I try:

`{"ReturnCode":"1001","ErrorMessage":"MyQ Application ID unrecognized. (1001)","CorrelationId":"8a0ac274-e2f0-4ba8-8baf-54fdb4d3c3f9"}`

Thanks!
